relayer/relayer.py

```diff
@@ -42,9 +43,31 @@ async def send_usdt(to_address, amount):
     return txn.broadcast().wait()
 
 async def is_profitable(spent, received):
-    # the swap is profitable only if the input is in USDC and USDT output is at least -0.2 from USDC amount
-    # this will be replaced with better rate approximation in the production relayer
-    return spent["token"].lower() == "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913".lower() and spent["amount"] - 200_000 >= received["amount"]
+    response = requests.get(config["api_url"] + "/assets")
```

Comment:
`/assets/{spent["token"]}`
Ill add an EP to support this

Reply:
I think we can just do this once on startup to have all decimals

Reply:
When you fully implement the backend, push the change in this branch